### ISSUE TITLE
Implement new designs for "Mark all as read" for Notifications panel

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#1312]
### Problem

See problem description in #1289.

### Solution

Implement ["Option 5"](https://github.com/okTurtles/group-income/issues/1289#issuecomment-1155295920) ([figma link](https://www.figma.com/file/mxGadAHfkWH6qApebQvcdN/Group-Income-2.0?node-id=23055%3A391024))

Leihla also suggests we should also separate the red counter from from the number of read messages, and have it be the count of notifications seen - and have the background of the message correspond to whether the message is actually "read".

In other words, when "Mark all as read" is clicked - it does two things:

1. It clears the counter entirely (all notifications are "seen")
2. It also changes the background of each notification by marking it as read.

And there are 2 ways to mark a notification as read:

1. Clicking "Mark all as read"
2. Clicking on the individual notification

**Comment by SebinSong** [comment]
Hi @taoeffect , @leihla 
I was playing around with the existing notification list UI components in the app and got a question.

Do you think [this notification modal](https://www.figma.com/file/mxGadAHfkWH6qApebQvcdN/Group-Income-2.0?node-id=4662%3A343) should contain "mark all as read" button in it too?

![image](https://user-images.githubusercontent.com/17641213/175809384-83f8d24e-7c7d-4f01-94c5-0f6196baf724.png)


Apparently on mobile/tablet, clicking on the notification bell button on the navigation menu directly pops this modal out, rather than displaying the tooltip dialog UI in ["Option 5"](https://github.com/okTurtles/group-income/issues/1289#issuecomment-1155295920)

If you agree on this and adding the design for it in the figma is not of a big work, I can work on implementing both at the same time once the design is ready.

Cheers,

**Comment by taoeffect** [comment]
@SebinSong I am not sure whether this modal is supposed to be shown when the bell is clicked on a tablet, I'd like to hear @leihla's opinion on that.

But yes, I think this modal should also contain a "Mark all as read" link somewhere. Please feel free to add it somewhere that makes sense to you, and then if Leihla thinks it should go somewhere else she can make designs for it.

**Comment by leihla** [comment]
@SebinSong @taoeffect 

I've updates the notifications page to have the "Mark all as read" in all devices. All should be ready to implement now.

**Comment by SebinSong** [comment]
Thanks @leihla :)
